core/lib/auth/auth/scopes.py

```diff
@@ -2,3 +2,4 @@
 PULSE3D_USER_SCOPES = frozenset(["pulse3d:free", "pulse3d:paid", "pulse3d:rw_all_data"])
 PULSE3D_SCOPES = frozenset([*CUSTOMER_SCOPES, *PULSE3D_USER_SCOPES])
 ACCOUNT_SCOPES = frozenset(["users:verify", "users:reset", "customer:reset"])
+PULSE3D_PAID_USAGE = {"pulse3d": {"uploads": -1, "jobs": -1}}
```

Comment:
Can we make this immutable? You could just json.dumps it to a string instead of using immutabledict if that's nicer

Reply:
json.dumps here? it also json dumps in the route?